src/celeritas/ext/GeantSetup.cc

```diff
@@ -34,50 +31,35 @@ namespace
 {
 //---------------------------------------------------------------------------//
 /*!
- * Set up global geant4 physics.
+ * Load the detector geometry from a GDML input file.
  */
-void load_physics(const GeantSetupOptions& options, G4RunManager* run_manager)
+class DetectorConstruction : public G4VUserDetectorConstruction
```

Comment:
Any reason to move the `DetectorConstruction` class inside `GeantSetup.cc` instead of keeping it separate---like `ActionInitialization`, `PrimaryGeneratorAction`, and so on?

Reply:
Only because I was modifying it, saw that:
- it's only used in one translation unit
- it's only a handful of lines
- it's not independently tested

The others you mention are likewise very small, so I'll just do the same :)